Create dedicated folder for the mod for Otto P1-90166 emer jett switch

eHanseJoerg opened the pull request:
# Description

This is an editorial change. I moved the modification for the Otto P-190166 emergency jettison switch into a dedicated folder. I also generated and provided a _MODIFICATIONS.md in that folder. 

Addresses # (issue)

Files in the OHM_modifications folder are currently not conforming to the structure mentioned in the folder's readme.md. This is the first step to move all modifications into a respective folder and therefore clean up the modification folder.

## Motivation and Context

This cleans up the OHM_Modifications folder and prepares it for a future push of the Winwing mod.

## Type of change

Please delete options that are not relevant.

- [X] Docs - New (A new document that is neither MCAD or ECAD)
- [X] Docs - Revision (A revision to an existing document that is neither MCAD or ECAD)

# How Has This Been Tested?

not required; only editorial improvement.

jrsteensen commented:
Thank you @eHanseJoerg for cleaning up my mess. I'll review as soon as I get the chance.

jrsteensen commented on release/OHM_Modifications/README.md:
```diff
@@ -1,6 +1,5 @@
 # Mods Folder
 
-All parts inside the "mods" folder are modifications of the original OpenHornet parts. All parts inside this folder do not belong to the OpenHornet release baseline and are therefore not officially supported by the OpenHornet team.
+The parts inside this folder are modifications from the original OpenHornet baseline, provided by community members. They provide optionalalternative ways to build parts of your OpenHornet pit. They do not belong to the OH baseline and are not supported by the OpenHornet team. 
```
Please revert the changes to line 3.

jrsteensen reviewed (changes requested):
Changed this to a draft, and a few comments:

1. Please follow the [manufacturing file naming conventions.](https://github.com/jrsteensen/OpenHornet/wiki/HOWTO:-Generating-Fusion360-Manufacturing-Files) naming each part as the part it replaces with the addition of the `_MOD_A` to the part number.
2. Add _MODIFICATIONS.md to your new folder describing which parts are replaced and how to make the modification.
3. Revert line 3 of the README.md to the original text.

Please let me know when it is ready for review and to be committed to the repo.

eHanseJoerg commented:
1. The commit fully adheres to the file naming conventions previously, including the MOD_A addendum to the filename.
2. A _MODIFICATIONS.md already exists in the folder, including description
3. Reverted
4. I don't know what that means: "changes required" . However, PR description will be updated.
5. Deleted

jrsteensen commented:
> 1. The commit fully adheres to the file naming conventions previously, including the MOD_A addendum to the filename.
> 2. A _MODIFICATIONS.md already exists in the folder, including description
> 3. Reverted
> 4. I don't know what that means: "changes required" . However, PR title and description updated.
> 5. Deleted

Apologies, that review was against when you had the PR including your winwing mod. They don't apply since you removed that from this PR. Thank you for the PR @eHanseJoerg!
